java/yb-pgsql/src/test/java/org/yb/pgsql/TestPgDelete.java

```diff
@@ -201,4 +201,41 @@ public void testDeleteReturn() throws SQLException {
       assertEquals(expectedRows, getSortedRowList(returning));
     }
   }
+
+  @Test
+  public void testDeleteUsing() throws SQLException {
+    String tableName1 = "test_delete_using";
+    String tableName2 = "test_helper";
+    createSimpleTable(tableName1);
+    createSimpleTable(tableName2);
+
+    // Fill in the helper table:
+    try (Statement insert_stmt = connection.createStatement()) {
+      insert_stmt.execute("INSERT INTO test_helper(h, r, vi, vs) VALUES(1, 0.5, 10, 'v')");
+    }
+    
+    List<Row> expectedRows = new ArrayList<>();
+    try (Statement insert_stmt = connection.createStatement()) {
+      String insert_format = "INSERT INTO %s(h, r, vi, vs) VALUES(%d, %f, %d, '%s')";
+      for (long h = 0; h < 5; h++) {
+        String insert_text = String.format(insert_format, tableName1,
+                                             h, h + 0.5, h * 10 + h, "v" + h );
```

Comment:
style: this is now overindented (match `(`)

Reply:
@jaki Thanks, I've made the final changes.